File: OPENAPI-REST-API/openapi-client/zapier/models/QsOrderRequest.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}server`,
                label: `QuickServer plan/server id from getNewQs. - [${labelPrefix}server]`,
                required: true,
                type: 'integer',
            },
            {
                key: `${keyPrefix}password`,
                label: `Root password for the QuickServer. - [${labelPrefix}password]`,
                required: true,
                type: 'string',
            },
            {
                key: `${keyPrefix}tos`,
                label: `Terms-of-service acceptance. Must be true to place the order. - [${labelPrefix}tos]`,
                required: true,
                type: 'boolean',
            },
            {
                key: `${keyPrefix}os`,
                label: `OS template file name from getNewQs (falls back to a default if unrecognized). - [${labelPrefix}os]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}comment`,
                label: `Free-form note saved on the service row. - [${labelPrefix}comment]`,
                type: 'string',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'server': bundle.inputData?.[`${keyPrefix}server`],
            'password': bundle.inputData?.[`${keyPrefix}password`],
            'tos': bundle.inputData?.[`${keyPrefix}tos`],
            'os': bundle.inputData?.[`${keyPrefix}os`],
            'comment': bundle.inputData?.[`${keyPrefix}comment`],
        }
    },
}

File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/QsOrderRequest.scala

package org.openapitools.client.model


case class QsOrderRequest (
    /* QuickServer plan/server id from getNewQs. */
    _server: Integer,
    /* Root password for the QuickServer. */
    _password: String,
    /* Terms-of-service acceptance. Must be true to place the order. */
    _tos: Boolean,
    /* OS template file name from getNewQs (falls back to a default if unrecognized). */
    _os: Option[String],
    /* Free-form note saved on the service row. */
    _comment: Option[String]
)
object QsOrderRequest {
    def toStringBody(var_server: Object, var_password: Object, var_tos: Object, var_os: Object, var_comment: Object) =
        s"""
        | {
        | "server":$var_server,"password":$var_password,"tos":$var_tos,"os":$var_os,"comment":$var_comment
        | }
        """.stripMargin
}
